SPECS/cloud-hypervisor-cvm/cloud-hypervisor-cvm.spec

```diff
@@ -17,12 +17,20 @@ Source0:        https://github.com/microsoft/cloud-hypervisor/archive/refs/tags/
 # To update the cache and config.toml run:
 #   tar -xf %{name}-%{version}.tar.gz
 #   cd %{name}-%{version}
+#   patch -u -p0 < ../upgrade-openssl-to-3.3.2-to-address-CVE-2024-6119.patch
```

Comment:
Please apply the patch in the `%prep` section. This helps to keep the flow consistent:
1. Extract unmodified source tarballs.
2. Patch their contents.

Also, this way we don't need to generate and upload a whole new tarball to the blob store.

Reply:
I dont think we can do that. My understanding is that we need to patch the Cargo.lock before cargo vendor, in order to cache the updated bits of the openssl. 

So I think we should leave it in the comment, upload the new vendor archive. But I will move it from %setup to %prep. 

Do you see any concern with this approach?

Reply:
Prep does not work as is, so will move back to setup.